src/packages/@app/pages/Dashboard/Dashboard.svelte

```diff
@@ -204,7 +204,7 @@
     <section class="w-100">
       <!-- Route for Collections -->
       <Route path="/collections/*">
-        <CollectionsPage />
+        <CollectionsPage {isLoginBannerActive} />
       </Route>
```

Comment:
not required to pass anything to collection page as  a props, as collection page has their own viewmodel to handle this.

Reply:
fixed